testsuite/smoke-test/src/aptos_cli/validator.rs

```diff
@@ -198,6 +198,7 @@ async fn test_onchain_config_change() {
             let inner = match genesis_config.consensus_config.clone() {
                 OnChainConsensusConfig::V1(inner) => inner,
                 OnChainConsensusConfig::V2(inner) => inner,
+                OnChainConsensusConfig::V3(inner) => inner.main,
```

Comment:
unrelated change

Reply:
it's kinda not, I was running this test to see the logs, and the test is broken (it's ignore), so I fixed it here :)